File: C/771_JewelsAndStones.c
#include <stdio.h>

int numJewelsInStones(char *jewels, char *stones)
{
	int res = 0;
	int x;
	int z;

	for (x = 0; jewels[x] != '\0'; x++)
	{
		for (z = 0; stones[z] != '\0'; z++)
		{
			if (jewels[x] == stones[z])
				res++;
		}
	}
	return (res);
}

File: C#/771_JewelsAndStones.cs
public class Solution
{
	public static int NumJewelsInStones(string jewels, string stones)
	{
		char[] arrJewels = jewels.ToCharArray();
		char[] arrStones = stones.ToCharArray();
		int res = 0;
		foreach (char j in arrJewels)
		{
			foreach (char s in arrStones)
			{
				if (j == s)
					res++;
			}
		}
		return (res);
	}
}
